builder.go

```diff
@@ -361,27 +358,45 @@ type FuncOption struct {
 	key     string
 }
 
-func Option(key string, operand any) FuncOption { return FuncOption{key: key, operand: operand} }
-
 func Func(name string, option ...FuncOption) *Expression {
 	return Expr().Func(name, option...)
 }
 
+func VarOption(name, varName string) FuncOption {
+	return FuncOption{key: name, operand: varSymbol + varName}
```

Comment:
We use `variable` type in the `func (e *Expression) Var(name string)` but not here. Why?

It should be consistent - we cannot use `string` in one place and `variable` in another for the same type.

```suggestion
	return FuncOption{key: name, operand: variable(varName)}
```